osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/SliderSelectionBlueprint.cs

```diff
@@ -240,10 +240,10 @@ private void updatePath()
             new OsuMenuItem("Add control point", MenuItemType.Standard, () => addControlPoint(rightClickPosition)),
         };
 
-        public override Vector2 ScreenSpaceSelectionPoint => BodyPiece.ToScreenSpace(BodyPiece.PathStartLocation);
+        public override Vector2 ScreenSpaceSelectionPoint => this.ToScreenSpace(slider.HitObject.StackedPosition);
```

Comment:
to resolve CI error:
```suggestion
        public override Vector2 ScreenSpaceSelectionPoint => ToScreenSpace(slider.HitObject.StackedPosition);
```


Not sure if it's better, but noticed you can rewrite this to `slider.Parent.ToScreenSpace(slider.DrawPosition)`

Reply:
i prefer the current method. the suggestion above is making assumptions that `slider.Parent` is non-null which seem unsafe given pooling - definitely less safe than referring to `this`.